CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix encoding and decoding of singular tuples and records (optimization)
 - Fix support of templated records
+- Fix omitting of optional record fields 
```

Comment:
Not sure about this, omitting of optional fields was broken after the previous release (at the same time as `Set` support)

Reply:
Yep, I found it while running sdk tests with the calldata from master branch